<launch>
  <arg name="sim" default="true"/>
  <arg name="use_tf" default="true"/>
  <arg name="robot_size" default="6"/>
  <arg name="dilate_probability" default="30"/>
  <rosparam param="use_tf" subst_value="true">$(arg use_tf)</rosparam>

  <include file="$(find asl_turtlebot)/launch/root.launch">
    <arg name="world" value="project_city5" />
    <arg name="x_pos" default="3.15"/>
    <arg name="y_pos" default="1.6"/>
    <arg name="z_pos" default="0.0"/>
    <arg name="rviz" default="section4"/>
    <arg name="model" default="asl_turtlebot"/>
    <arg name="sim" default="$(arg sim)"/>
  </include>

  <node pkg="asl_turtlebot" type="map_dilator.py" name="map_dilator" output="screen">
    <param name="robot_size" value="$(arg robot_size)"/>
    <param name="dilate_probability" value="$(arg dilate_probability)"/>
  </node>
</launch>
